src/mixxx.cpp

```diff
@@ -1113,6 +1113,25 @@ void MixxxMainWindow::initActions()
     m_pFileQuit->setWhatsThis(buildWhatsThis(quitTitle, quitText));
     connect(m_pFileQuit, SIGNAL(triggered()), this, SLOT(slotFileQuit()));
 
+    
+    QString player3LoadStatusText = loadTrackStatusText.arg(QString::number(3));
```

Comment:
Please put these blocks for player 3/4 immediately after similar blocks for deck 1/2 and before block for File/Quit